### DISCUSSION TITLE
[QA] -> by Update or fresh install, checkbox Start/Restart Open-Shell (or check the automatic start at the Setup)

### DISCUSSION THREAD

**Discussion by blackcrack** [Open-Shell/Open-Shell-Menu#1129] [Q&A]
[[QA] -> by Update or fresh install, checkbox Start/Restart Open-Shell (or check the automatic start at the Setup)](https://github.com/Open-Shell/Open-Shell-Menu/issues/1112)

Windows 7 Professional
all Updates installed up the last update
4.4.177

At Update from OpenShell
explorer stops and restarts,
but OpenShell restarts not !

have anyone the same behaver at OpenShellSetup_4_4_177.exe

best

**Comment by CTVCAM8** [comment]
Yes.

Most Open-Shell updates on Windows 10 don't go as smoothly as Classic Shell updates in the past.
This could be due to lots of beta installs?

I recall after 4.4.171 finished upgrading, 
I had to do a system restart to get the start button to show up.
I think the installer had trouble shutting everything down 
in explorer and then starting it up again.
Whenever that window asked to "close the applications and attempt to restart them" 
or "Not to close applications. A reboot will be required."  
I have always picked the default option "Close the applications and attempt to restart them."

![Default option](https://user-images.githubusercontent.com/44127216/190011456-aaddf0c0-289a-46f1-bade-96716d97d3f2.png)

Test1 OpenShellSetup_4_4_177.exe:
After I get through Windows 10 security hurdles, I start the installation process.
I picked the default option "Close the applications and attempt to restart them."
Setup finishes.
Problem. 
Clicking the start button opens Win 10 start menu not Open_Shell.
I check Task manager to see if Open-Shell is running- no.
I also notice that "Microsoft Compatibility Telemetry" is running with 80% CPU.
After it goes away still no Open-Shell start menu.
Open-Shell shows in the startup tab of Task manager though.
I open the Win10 start menu again, 
expand the Open-Shell folder and click on "Open-Shell Menu Settings"
and Open-Shell 4.4.177 start menu is working and shows up in the Task monitor. 



Test2 OpenShellSetup_4_4_178:
After I get through Windows 10 security hurdles, I start the installation process.
I picked the default option "Close the applications and attempt to restart them."
Setup finishes.
Success.
I click the start button and get Open-Shell 4.4.178
This time I noticed "Microsoft Compatibility Telemetry" was running
with 25% CPU and then went away after about a minute.
Probably has nothing to do with it, but mentioned it anyway. 

So, what does all the above tell me besides now being up to date?
The default option does not always work (less than 50% success).

Maybe we should always use the other option to reboot instead?
